_episodes/08-defensive.md

```diff
@@ -531,11 +531,13 @@ This violates another important rule of programming:
 > >
 > > def range_overlap(ranges):
 > >     '''Return common overlap among a set of [low, high] ranges.'''
+> >     if not ranges:
```

Comment:
This is true, but not a situation I suspect our novice students will run into as I'm not sure we ever talk about `None` in the lesson. The more likely situation is they call it and pass nothing, which raises a `TypeError`. That said, if you strongly prefer to keep it this way I can probably live with it.